src/main/java/io/fabric8/jenkins/openshiftsync/GlobalPluginConfiguration.java

```diff
@@ -114,6 +116,28 @@ private void configChange() {
         OpenShiftUtils.initializeOpenShiftClient(server);
         this.namespace = getNamespaceOrUseDefault(namespace, getOpenShiftClient());
 
+        logger.info("Waiting for Jenkins to be started");
```

Comment:
Why do you have to wait until Jenkins is ready for this task? Does it not suffice to wait until all jobs are loaded? See http://javadoc.jenkins.io/hudson/model/listeners/ItemListener.html#onLoaded()